Index: src/Book.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>public class Book {\r\n    private String bookTitle;\r\n    private int bookPrice;\r\n    private int releaseData;\r\n\r\n    public Book(String bookTitle, int bookPrice, int releaseData) {\r\n        this.bookTitle = bookTitle;\r\n        this.bookPrice = bookPrice;\r\n        this.releaseData = releaseData;\r\n    }\r\n\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Book.java b/src/Book.java
--- a/src/Book.java	(revision 234a44795b52eec22baac8633ba8bd7e33ba5825)
+++ b/src/Book.java	(date 1750707396645)
@@ -8,5 +8,4 @@
         this.bookPrice = bookPrice;
         this.releaseData = releaseData;
     }
-
 }
